### ISSUE TITLE
implement token for accesible border width

### ISSUE THREAD

**Issue by lovemecomputer** [IBM/carbon-components#2323]
on tabs/etc that use a border to highlight the current location, the border should be a minimum of `3px` to meet accessibility standards. this is a a thing i've had to implement in a couple places on the website. i recommend making a new token for this width, and then doing a pass thru all of our components to make sure any 2px borders are bumped up to this

**Comment by asudoh** [comment]
Makes sense to me, @IBM/carbon-developers any thoughts? Also should this be in `carbon-elements`?
